tests/unit/RequestTest.php

```diff
@@ -3,31 +3,31 @@
 
 namespace WMDE\Fundraising\Store\Tests;
 
-use WMDE\Fundraising\Entities\Request;
+use WMDE\Fundraising\Entities\Inquiry;
 
 class RequestTest extends \PHPUnit_Framework_TestCase
```

Comment:
Test class could have been renamed too.
Oh no, and I have just noticed that the bracketing is not right here (not a problem with this change, though)